crates/typed-store/src/rocks/mod.rs

```diff
@@ -308,8 +358,17 @@ where
 
     #[instrument(level = "trace", skip_all, err)]
     fn get_raw_bytes(&self, key: &K) -> Result<Option<Vec<u8>>, TypedStoreError> {
+        let _timer = self
```

Comment:
Again the timer here is probably unnecessary.  Although, we can rename the span name to make it more unique.

Reply:
same